src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/NotSuitableSDOCallbackHandler.java

```diff
@@ -60,6 +65,19 @@ private CallbackResponse submitNotSuitableSDO(CallbackParams callbackParams) {
             .build();
     }
 
+    private CallbackResponse validateNotSuitableReason(CallbackParams callbackParams) {
+        List<String> errors = new ArrayList<>();
+        var reason = callbackParams.getRequest().getCaseDetails().getData().get("reasonNotSuitableSDO");
+        if (reason.toString().length() > lengthAllowed + "{input=}".length()) {
```

Comment:
lengthAllowed + "{input=}".length() This appears to be constant ? Just make it a constant in that case

Reply:
lengthAllowed is a constant already, but "{input=}".length() can become another

Reply:
"{input=}".length() is no longer necessary as I reach the input variable directly now via casedata rather than casedetails.